bin/tigmint_molecule.py

```diff
@@ -0,0 +1 @@
+tigmint-molecule
```

Comment:
I understand that you've made this soft link so you're able to import the module into `tigmint_molecule_paf.py` while sticking to the naming convention -- I'm wondering if it's more straightforward to rename the existing scripts to snake case so this isn't necessary? That way you wouldn't have to make a softlink to `tigmint_molecule_paf.py` to match the file names either. What do you think?

Reply:
Yeah you're right - that was my workaround so I could do that import
I think you're right - it's getting messy with the multiple soft-links - I'll rename the files so that we don't need any of those soft-links!